ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -84,9 +90,10 @@ internal fun Markdown(
 
     Column(
         verticalArrangement = Arrangement.spacedBy(paragraphPadding),
+        horizontalAlignment = horizontalAlignment,
```

Comment:
We are also passing the horizontal alignment to the modifier, but I had to pass it here as well. Reason is that, if the content size is set to `Fill`, the horizontal alignment was previously mostly ignored. I think this might be a better place for it. I thought of removing it from the modifier, but left it for now just in case